src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.targets

```diff
@@ -274,13 +274,17 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <PropertyGroup>
       <DefaultItemsMoreInformationLink>https://aka.ms/sdkimplicititems</DefaultItemsMoreInformationLink>
+    </PropertyGroup>
+
+    <!-- NOTE for design-time builds we continue on errors and remove the duplicate items.
+         This is because otherwise there won't be any references to pass to the compiler, leading to design-time
+         compilation errors for every API that is used in the project.  Amidst all the compile errors, it would
+         be easy to miss the duplicate items error which is the real source of the problem. -->
 
-      <!-- For the design-time build, we will continue on error and remove the duplicate items.
-           This is because otherwise there won't be any references to pass to the compiler, leading to design-time
-           compilation errors for every API that is used in the project.  Amidst all the compile errors, it would
-           be easy to miss the duplicate items error which is the real source of the problem. -->
-      <CheckForDuplicateItemsContinueOnError>false</CheckForDuplicateItemsContinueOnError>
-      <CheckForDuplicateItemsContinueOnError Condition="'$(DesignTimeBuild)' == 'true'">ErrorAndContinue</CheckForDuplicateItemsContinueOnError>
+    <!-- We no longer use the CheckForDuplicateItemsContinueOnError property here, but the WPF repo has a dependency upon it.
+         Remove this assignment if https://github.com/dotnet/wpf/pull/1235 is merged. -->
```

Comment:
Good catch. I'll merge this now. Let's wait until a few days after WPF merges to remove this, just to avoid a race getting to the core-sdk repo between sdk and wpf and having some bad builds.